FRO5401/ChargedUp2023
# Removing un-used code

## Description

- Removing code that is unused in classes
- Removing unused code imports

## Related Issue
#12 

## Motivation and Context

- To make the robot code less bloated, I began to sweep through the code to find obsolete code, and variables that are declared and initialized, but never used outside of a constructor.  This will help make it easier to read, understand, and ultimately run more efficiently on the RoboRio.

## Changes Made

- Code imports
- Unused variables outside of the constructor

## Screenshots (if applicable)


## How to Test

- Build & deploy code to the Bucky
- Perform a full systems check of all systems
- Test Auto

## Checklist


- [ x ] The code follows the project's coding guidelines
- [ x ] New tests have been added to cover the changes (if applicable)
- [ x ] Documentation has been updated to reflect the changes (if applicable)

## Additional Notes
@Ace-h121 Please ensure the new programmers view this PR and understand the interface
